packages/usa-in-page-navigation/src/index.js

```diff
@@ -187,20 +187,29 @@ const getSectionId = (value) => {
   return id;
 };
 
+/**
+ * Recursively calculate a heading element's offset from the top of the page.
+ *
+ * @param {HTMLElement} el A heading element
+ */
+const getTotalElementOffset = (el) =>
+  el.offsetTop + (el.offsetParent && getTotalElementOffset(el.offsetParent));
+
 /**
  * Scroll smoothly to a section based on the passed in element
  *
- * @param {HTMLElement} - Id value with the number sign removed
+ * @param {HTMLElement} el A heading element
```

Comment:
**polish:** It looks like the `el` here is actually the anchor element if we want to specify

```suggestion
 * @param {HTMLAnchorElement} el - The link to the section heading.
```